Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -3032,6 +3906,28 @@ public void Should_fail_with_descriptive_message_when_asserting_string_length_to
 
         #endregion
 
+        [Fact]
+        public void When_chaining_multiple_contain_assertions_it_should_assert_all_conditions()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            string actual = "ABCDEFGHI";
+            string substring = "EF";
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act / Assert
+            //-----------------------------------------------------------------------------------------------------------
+            actual.Should()
```

Comment:
🔧 I'd like to be able to write
```c#
actual.Should().Contain(substring).AtLeast.Once().And.AtMost.Twice();
```
Right now `CountOccurrences is re-computed` for every call to `Contain()`?

Reply:
Ok, I can change it.